sdk/communication/azure-communication-phonenumbers/pom.xml

```diff
@@ -63,7 +63,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-communication-common</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
+      <version>1.0.3</version> <!-- {x-version-update;com.azure:azure-communication-common;current} -->
```

Comment:
not needed

Reply:
But this will fail the analyzer step since 1.1.0-beta.1 is not current. That's the error I was getting in Chat